src/shogun/lib/StoppableSGObject.cpp

```diff
@@ -40,6 +41,16 @@ rxcpp::subscription CStoppableSGObject::connect_to_signal_handler()
 	return get_global_signal()->get_observable()->subscribe(subscriber);
 }
 
+void CStoppableSGObject::serialize_machine()
```

Comment:
This is for the user chossing to serialize during pause right?

Reply:
yes but we could use it anywhere else too... easy serialization method